src/workers/newNotificationV2Mail.ts

```diff
@@ -78,6 +78,7 @@ export const notificationToTemplateId: Record<NotificationType, string> = {
   squad_public_approved: '41',
   post_bookmark_reminder: '',
   streak_reset_restore: '',
+  squad_featured: '', // TODO: MI-514 - Add template ID and relevant test cases
```

Comment:
We are gonna add it?
Should we wait until we have it or doesn't matter?

Reply:
I will send a message to the product channel once we have the featured page ready for viewing as we will redirect the user there. Until then, we should wait before merging.